crates/nu-parser/tests/test_parser.rs

```diff
@@ -2397,3 +2397,56 @@ mod operator {
         );
     }
 }
+
+mod record {
+    use super::*;
+
+    use nu_protocol::ast::RecordItem;
+
+    #[rstest]
+    #[case(b"{ :: x }", "Invalid literal")] // Key is bare colon
+    #[case(b"{ a: x:y }", "Invalid literal")] // Value is bare word with colon
+    #[case(b"{ a: x('y'):z }", "Invalid literal")] // Value is bare string interpolation with colon
```

Comment:
Just to clarify we don't have bare string interpolation, so that may need some clarification

Reply:
I'm not sure what you mean here. By "bare string interpolation" I mean things like this: `{a: a(1 + 1)b}`, which is valid and produces the same record as `{a: 'a2b'}`. Am I not using the right terminology?

Reply:
huh, I thought that would not work in other contexts?